warehouse/query-core/src/test/java/datawave/query/testframework/AbstractFunctionalQuery.java

```diff
@@ -139,7 +138,9 @@ protected ShardQueryLogic createQueryLogic() {
     }
     
     @Before
-    public void querySetUp() {
+    public void querySetUp() throws IOException {
+        temporaryFolder = Files.createTempDirectory("tmp");
```

Comment:
Does this folder ever get cleaned up?

Reply:
Thanks for spotting that.  I need an @After function where I do that.  I did this in two other classes so it got a little confusing.

Reply:
It seemed get wiped out anyways.

Reply:
Pushed the code fix. I hit the ' temporaryFolder.toFile().deleteOnExit();' line when I put a breakpoint on it in the Intellij debugger and run IndexQueryLogicTest which uses AbstractFunctionalQuery.